src/SFA.DAS.TokenService.Infrastructure/Services/OAuthTokenService.cs

```diff
@@ -21,12 +21,12 @@ public OAuthTokenService(IHttpClientWrapper httpClient, OAuthTokenServiceConfigu
             _httpClient.AcceptHeaders.Add(new System.Net.Http.Headers.MediaTypeWithQualityHeaderValue("application/vnd.hmrc.1.0+json"));
         }
 
-        public async Task<OAuthAccessToken> GetAccessToken(string clientSecret)
+        public async Task<OAuthAccessToken> GetAccessToken(string privilegedAccessToken)
```

Comment:
This method returns the privilegedAccessToken right? The parameter is more like a one time secret

Reply:
Renamed references to oneTimePassword in 1188e84